plugins/ksp/testData/api/referenceElement.kt

```diff
@@ -0,0 +1,38 @@
+// WITH_RUNTIME
+// TEST PROCESSOR: ReferenceElementProcessor
+// EXPECTED:
+// KSClassifierReferenceImpl: Qualifier of B is A
+// KSClassifierReferenceImpl: Qualifier of C is A
+// KSClassifierReferenceImpl: Qualifier of Int is null
+// KSClassifierReferenceImpl: Qualifier of String is null
+// KSClassifierReferenceDescriptorImpl: Qualifier of B is A
+// KSClassifierReferenceDescriptorImpl: Qualifier of C<Int> is A<String>
+// KSClassifierReferenceDescriptorImpl: Qualifier of Int is null
+// KSClassifierReferenceDescriptorImpl: Qualifier of String is null
+// KSClassifierReferenceJavaImpl: Qualifier of H is J
+// KSClassifierReferenceJavaImpl: Qualifier of I is J
+// KSClassifierReferenceJavaImpl: Qualifier of Object is null
```

Comment:
Where is this Object from in type references?

Reply:
Implicit super class of `J`